src/utils/intl.ts

```diff
@@ -104,3 +104,20 @@ export function formatText(
 
   return message;
 }
+
+// Handle labels that are either strings or message descriptors
+export const useFormattedText = () => {
```

Comment:
I am not 100% sure of this one, but I think we could just use `formatText` here? cc @rdig, you know more of the inner workings of these helpers :)